Makefile

```diff
@@ -91,12 +91,15 @@ test:
 	export LOG_LEVEL=DEBUG && \
 	export RUST_BACKTRACE=1 && \
 	cargo test --features "${ENABLE_FEATURES}" --all ${EXTRA_CARGO_ARGS} -- --nocapture && \
-	cargo test --features "${ENABLE_FEATURES}" --bench misc ${EXTRA_CARGO_ARGS} -- --nocapture  && \
+	cargo test --features "${ENABLE_FEATURES}" --bench misc ${EXTRA_CARGO_ARGS} -- --nocapture && \
```

Comment:
You can use the same command like so: `cargo test --features "${ENABLE_FEATURES}" --bench misc --bench raftstore --bench engines ... ${EXTRA_CARGO_ARGS} -- --nocapture`